cs/test/core/JsonSerializationTests.cs

```diff
@@ -10,7 +10,7 @@
     class JsonSerializationTests
     {
         [Test]
-        public void Null_NonNullableStrings()
+        public void JsonSerialization_Null_NonNullableStrings()
```

Comment:
Minor: Move to JSON commit instead? Or should we merge to master as a single commit?